File: wls-eai-service/src/main/java/de/muenchen/oss/wahllokalsystem/eaiservice/rest/wahlvorschlag/model/Wahlvorschlag.java
package de.muenchen.oss.wahllokalsystem.eaiservice.rest.wahlvorschlag.model;

import io.swagger.v3.oas.annotations.media.Schema;
import jakarta.validation.constraints.NotNull;
import java.util.Set;

public record Wahlvorschlag(@NotNull String identifikator,
                            @Schema(requiredMode = Schema.RequiredMode.REQUIRED) long ordnungszahl,
                            @NotNull String kurzname,
                            @Schema(requiredMode = Schema.RequiredMode.REQUIRED) boolean erhaeltStimmen,
                            Set<Kandidat> kandidaten) {
}

File: wls-gui-wahllokalsystem/src/types/wahlvorschlaege/Wahlvorschlag.ts
import type { Kandidat } from "@/types/wahlvorschlaege/Kandidat.ts";

export interface Wahlvorschlag {
  identifikator: string;
  ordnungszahl: number;
  kurzname: string;
  erhaeltStimmen: boolean;
  kandidaten?: Kandidat[];
}
